rpm/qgis.spec.template

```diff
@@ -248,7 +253,10 @@ gzip ChangeLog
       -D WITH_CUSTOM_WIDGETS:BOOL=TRUE \
       -D BINDINGS_GLOBAL_INSTALL:BOOL=TRUE \
       -D ENABLE_TESTS:BOOL=FALSE \
+      -D WITH_EPT:BOOL=TRUE \
+      -D WITH_PDAL:BOOL=TRUE \
       -D WITH_QSPATIALITE:BOOL=TRUE \
+      -D WITH_QWTPOLAR:BOOL=TRUE \
```

Comment:
Fedora has prebuilt qwtpolar libs, I think the system libraries should be preferred?
CC @daniviga